SignalServiceKit/src/Messages/TSMessagesManager.m

```diff
@@ -42,6 +42,32 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#define kOWSProdAssertParameterEnvelopeIsLegacy @"envelope_is_legacy"
+#define kOWSProdAssertParameterEnvelopeHasContent @"has_content"
+#define kOWSProdAssertParameterEnvelopeDescription @"envelope_description"
+#define kOWSProdAssertParameterEnvelopeEncryptedLength @"encrypted_length"
+
+#define AnalyticsParametersFromEnvelope(__envelope)                                                                    \
+    ^{                                                                                                                 \
+        NSData *__encryptedData = __envelope.hasContent ? __envelope.content : __envelope.legacyMessage;               \
+        return (@{                                                                                                     \
+            kOWSProdAssertParameterEnvelopeIsLegacy : @(__envelope.hasLegacyMessage),                                  \
+            kOWSProdAssertParameterEnvelopeHasContent : @(__envelope.hasContent),                                      \
+            kOWSProdAssertParameterEnvelopeDescription : [self descriptionForEnvelopeType:__envelope],                 \
```

Comment:
This reads a little goofy. Key is for `description` but value is the envelope type. Maybe the key should be `kOWSProdAssertParameterEnvelopeType`.